databricks/koalas/groupby.py

```diff
@@ -422,6 +423,81 @@ def size(self):
                                              for i, s in enumerate(groupkeys)])
         return _col(DataFrame(internal))
 
+    def diff(self, periods=1):
+        """
+        First discrete difference of element.
+
+        Calculates the difference of a DataFrame element compared with another element in the
+        DataFrame (default is the element in the same column of the previous row).
+
+        .. note:: the current implementation of diff uses Spark's Window without
```

Comment:
If we use partition columns, the data will be distributed by its partition columns. So we could remove this note.

Reply:
@RainFung, looks this comment and the comment below were not addressed.